src/constants/TV.ts

```diff
@@ -66,3 +66,22 @@ export enum Keys {
   volumeUp = 'volumeup',
   yellowButton = 'yellowbutton',
 }
+
+export enum Apps {
```

Comment:
Please sort this and make it the first file export as it should appear before `export enum EnergySavingLevels {`.